README.md

```diff
@@ -68,7 +85,9 @@ docker build -t firecloud-orch .
 Run the container:
 ```
 docker run --name orch \
+  -p 8080:8080
+  -e GOOGLE_CLIENT_ID='$GOOGLE_CLIENT_ID' \
```

Comment:
This puts (or something else does) the following code into the html that is generated when going to http://dhost:8080/
```
<meta name="google-signin-client_id" content="$GOOGLE_CLIENT_ID">
```
I'm pretty sure that is not correct.